clearml_agent/commands/worker.py

```diff
@@ -2919,7 +2922,11 @@ def install_requirements_for_package_api(
          """
         if package_api:
             package_api.cwd = cwd
-        api = self._install_poetry_requirements(repo_info)
+
+        lockfile_path = execution.working_dir if package_api.session.config["agent.package_manager.poetry_files_from_cwd"] \
```

Comment:
@nielstenboom I think the name might be a bit misleading, as this option basically means "take the poetry files from the repo's configured working dir and not from the root" - perhaps `poetry_files_from_repo_working_dir` is a bit more clear?

Reply:
Another thing, for readability and safety, (and code organization) I would suggest structuring it like so:
```python
files_from_working_dir = package_api.session.config.get("agent.package_manager.poetry_files_from_cwd", False)
lockfile_path = Path(repo_info.root) / (execution.working_dir if files_from_working_dir else "")
...
```

Reply:
Good suggestions, I agree!